bsip-0062.md

```diff
@@ -0,0 +1,44 @@
+ ```
+  BSIP: 0062
+  Title: Close short position 
```

Comment:
This title is not good.

The BSIP is talking about a new option or operation to sell a debt position's collateral directly to pay the debt. The title should reflect the content.

Reply:
Do you have an alternative suggestion? I find it still suitable as it is looking to exactly do this.